content/en/docs/reference/access-authn-authz/certificate-signing-requests.md

```diff
@@ -304,7 +304,13 @@ kubectl create rolebinding developer-binding-john --role=developer --user=john
 
 The last step is to add this user into the KubeConfig. We assume the key and crt files are located here "/home/vagrant/work/".
 
-First, we need to add new credentials
+First, export the issued certificate from the CertificateSigningRequest:
```

Comment:
would this make sense to move to the `Get the Certificate` step above?

Reply:
makes sense